src/node/internal/streams_util.d.ts

```diff
@@ -1,3 +1,202 @@
+import { Buffer } from 'node:buffer';
```

Comment:
For this file, I pasted the original and had AI generate the typing file.

Reply:
Buffer is global. This line is not needed

Reply:
Yeah, I'm just still hopeful we can encourage this pattern as opposed to discouraging it in the Node.js ecosystem.